uk.co.compucorp.civicrm.tasksassignments/CRM/Activity/Service/ActivityService.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+class CRM_Activity_Service_ActivityService {
+
+  /**
+   * Returns the activity types for a certain component. Default limit is 25
+   *
+   * @param $componentName
+   *  The name of the component
+   * @param int $limit
+   *  The maximium number of results to return
+   *
+   * @return array
+   */
+  public static function findByComponent($componentName, $limit = 0) {
```

Comment:
The default is 0 because it's used in an API call, so 0 is unlimited. I've updated the description to make this clearer